Examples/Tests/collision/inputs_3d

```diff
@@ -64,6 +64,13 @@ collision2.ndt = 10
 collision3.ndt = 10
 
 # Diagnostics
-diagnostics.diags_names = diag1
+diagnostics.diags_names = diag1 diag_filter
 diag1.intervals = 10
 diag1.diag_type = Full
+
+## diag_filter is another diag used to test the particle filter function.
+diag_filter.intervals = 150:150:
```

Comment:
That way we don't check the output of this diag in a checksum, so we would not notice bugs that break the filter.
Maybe we can pick an example where we change the primary `diag1`?

Reply:
Actually, the laser-ion example now checks the plotfile particle filter since #1632 :)
https://github.com/ECP-WarpX/WarpX/blob/development/Examples/Physics_applications/laser_ion/inputs

Reply:
@ax3l The runtime results are tested in the analysis scripts. That should be ok, right?